src/states/features/product/productService.js

```diff
@@ -0,0 +1,18 @@
+import API from '../../api';
+import { API_URL, LOCAL_API_URL } from '../../../constants';
+
+const addProduct = async (data) => {
+  const token = localStorage.getItem('myToken');
+  const response = await API.post(`${API_URL}/products`, data, {
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `token=${token}`
+    }
```

Comment:
The content-type is set by default when you are using axios and the Authorization is set in the intercept that is in the api.js file, which do you have to set it again?